helm_deploy/hmpps-interventions-service/reports/team_kpi_details_ages.sql

```diff
@@ -11,7 +10,5 @@ FROM referral r
          JOIN intervention i ON r.intervention_id = i.id
          JOIN dynamic_framework_contract c ON i.dynamic_framework_contract_id = c.id
          JOIN contract_type ct ON c.contract_type_id = ct.id
-         LEFT JOIN pcc_region pr ON c.pcc_region_id = pr.id
-         LEFT JOIN nps_region nr ON (c.nps_region_id = nr.id OR pr.nps_region_id = nr.id)
-GROUP BY ct.name, nr.name
-ORDER BY ct.name, nr.name;
+GROUP BY 1
```

Comment:
do these actually do anything? is the behaviour the same without them?

Reply:
yes, we need to specify group by as percentiles are aggregate functions and `ct.name` is the 'fact' it aggregates around